scripts/test-ui-testrunner.sh

```diff
@@ -8,13 +7,16 @@ export $(egrep -v '^#' .env | xargs) > /dev/null 2>&1
 npx start-storybook -p $TEST_HOST_STORYBOOK_PORT -c .storybook -s ./src/static --ci --quiet > /dev/null 2>&1 &
 npx wait-on $TEST_HOST_STORYBOOK_URL -t 30000
 
-if [ -z "$1" ]
-then
-    npx testcafe -c ${NUM_CPUS} "chrome:headless" -q ./**/ui.test.js
-else
-    npx testcafe -F "${1}" -c ${NUM_CPUS} "chrome:headless" -q ./**/ui.test.js
+sh ./scripts/test-playwright.sh
```

Comment:
`npx jest --config=jest.ui.config.js` here?

Reply:
imho only if we also inline the testcafe script.